Metabase Analysis.R

```diff
@@ -1009,3 +1014,51 @@ plhdata_org_clean %>% select('app_user_id', "rp.contact.field.user_age")
 plhdata_org_clean %>% select('app_user_id', "rp.contact.field.user_gender")
 plhdata_org_clean %>% filter(Org == "Amathuba") %>% select('app_user_id', "rp.contact.field.user_gender")
 mean(x=as.numeric(plhdata_org_clean$rp.contact.field.user_age), na.rm=TRUE)
+
+##################################
+##################################
+### Notification Data Analysis ###
+##################################
+##################################
+
```

Comment:
@MargheritaPhilipp I have added in the notification data analysis here.

One point is this: The nf data has 277 rows initially. However, after we filter the rows to only consider users in the plhdata_org_clean data set, we have eight rows.
This is (mainly) because 87% of users have a "Missing" organisation.

In the PLH Data, there are 1128 rows who are missing an organisation. Only 167 are not missing an organisation. We are throwing away *a lot* of data!

I think we should discuss the missing organisation situation, perhaps with @volloholic?, we are throwing a lot of data away over this, and I'm not convinced that's sensible!